python/py-scikit-rf/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        scikit-rf scikit-rf 0.15.3 v
+name                py-scikit-rf
+
+checksums           rmd160  33172e881c9b13d464a3e4bf0a5329c60ac1b188 \
+                    sha256  c0f84010eec1ac07755c9fb8e51295a0116037f2de0a2117f507f7f1848106da \
+                    size    17777889
+
+platforms           darwin
```

Comment:
the port doesn't install any architecture-dependent files, so please add the line `supported_archs noarch` underneath the `platforms` line